restli-int-test/src/test/java/com/linkedin/restli/examples/TestFillInDefaultValueFooDefault.java

```diff
@@ -0,0 +1,128 @@
+package com.linkedin.restli.examples;
+
+import com.linkedin.data.DataList;
+import com.linkedin.r2.RemoteInvocationException;
+import com.linkedin.restli.client.FindRequest;
+import com.linkedin.restli.client.GetAllRequest;
+import com.linkedin.restli.common.CollectionMetadata;
+import com.linkedin.restli.common.CollectionResponse;
+import com.linkedin.restli.common.Link;
+import com.linkedin.restli.common.LinkArray;
+import com.linkedin.restli.common.RestConstants;
+import com.linkedin.restli.examples.defaults.api.FooDefault;
+import com.linkedin.restli.examples.defaults.api.FooDefaultFillingRequestBuilders;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+
+public class TestFillInDefaultValueFooDefault extends RestLiIntegrationTest
```

Comment:
Rename this to
`TestFillInCollectionMetadataDefaultValue`

Reply:
End up augmenting existing test!